app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/newConversationModal.tsx

```
@@ -223,60 +223,47 @@ const CcAndBccInfo = ({
   onChange: (info: Partial<NewConversationInfo>) => void;
   onModEnter?: () => void;
 }) => {
-  const [ccVisible, setCcVisible] = useState(false);
-  const [bccVisible, setBccVisible] = useState(false);
+  const [showCcBcc, setShowCcBcc] = useState(
+    newConversationInfo.cc.length > 0 || newConversationInfo.bcc.length > 0
+  );
```

ESLint

[error] Replace `⏎····newConversationInfo.cc.length·>·0·||·newConversationInfo.bcc.length·>·0⏎··` with `newConversationInfo.cc.length·>·0·||·newConversationInfo.bcc.length·>·0`

(prettier/prettier)